zookeeper-server/src/main/java/org/apache/zookeeper/client/FourLetterWordMain.java

```diff
@@ -94,6 +94,33 @@ public static String send4LetterWord(
         return send4LetterWord(host, port, cmd, secure, timeout, null);
     }
 
+    /**
+     * Send the 4letterword
+     * @param host the destination host
+     * @param port the destination port
+     * @param cmd the 4letterword
+     * @param timeout in milliseconds, maximum time to wait while connecting/reading data
+     * @param clientConfig client config
+     * @return server response
+     * @throws SSLContextException
+     * @throws IOException
+     */
+    public static String send4LetterWord(
+            String host,
+            int port,
+            String cmd,
+            int timeout,
+            ZKClientConfig clientConfig) throws SSLContextException, IOException {
```

Comment:
Keep `timeout` the last to consistent with `String send4LetterWord(String host, int port, String cmd, boolean secure, int timeout)` ?